nb2workflow/galaxy.py

```diff
@@ -0,0 +1,591 @@
+
+import xml.etree.ElementTree as ET
+import os
+import re
+from textwrap import dedent
+import argparse
+import logging
+
+import yaml
+import json
+
+from oda_api.ontology_helper import Ontology
+from nb2workflow.nbadapter import NotebookAdapter, find_notebooks
+
+import nbformat
+from nbconvert.exporters import ScriptExporter
+
+from ensureconda.api import ensureconda
+import subprocess as sp
+
+import bibtexparser as bib
+import pypandoc
+
+import tempfile
+
+logger = logging.getLogger()
+        
+               
+default_ontology_path = 'http://odahub.io/ontology/ontology.ttl'
+
+global_req = ['ipython']
+
+_success_text = '*** Job finished successfully ***'
+
+class GalaxyParameter:
```

Comment:
there was this package suggested by Bjoern, right? Even if you do not use it, could you put it in comment for the reference?